src/FileLoader.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Contributte\Neonizer;
+
+use Contributte\Neonizer\Config\FileConfig;
+use Contributte\Neonizer\Decoder\IDecoderFactory;
+
+class FileLoader
```

Comment:
Well, why do you add this class? I mean, there's no interface, there's no any other implementation. Maybe, I would do it more simple and keep it the old way, WDYT?

Reply:
To prevent code duplication; this file loading bit is now used not only by FileProcessor, where the code originally was, but also by FileValidator, so I extracted it into a separate class. I'm open to ideas how to do it any other old way.

I personally tend not to introduce interfaces when there is only one implementation because it's unnecessary and can be easily abstracted later when the need arises. But I'm not against the idea, if you'd like me to add an interface for this class.